### ISSUE TITLE
Feature Suggestion: Nutrients vs Power

### ISSUE THREAD

**Issue by Badcholo** [simeonradivoev/MatterOverdrive#299]
Becoming an android removes a difficulty. It makes it too easy to just not ever have to eat or breathe.
Making consumption of increased nutrients while under water will make scuba gear and underwater breathing enchantments still useful.
Infinite power and an android only requiring power makes it too easy.

Suggestion: A machine that turns food into a Nutrient liquid.
A spot to place a tank with nutrients to keep a player healthy using the android station next to the battery.

It would make food useful and more complicated foods even better cause they would produce more nutrients depending on their hunger and saturation. Possibly mixed in with a metal to produce the nanites that help heal the player with the ability.

**Comment by shadoxxhd** [comment]
i think androids should run exclusively on energy. but you could make them able to consume food to refill their energy reserves. and what should an android need oxigen for (or any other part of the air, for that matter). borg however could have an upgrade path leading to them needing food in order to stay alive, although they should have an option to run on electricity (2 tech trees: geneering and implants/nanonics (maybe a 3rd, endgame one which is soething like a crossover of the previous 2; last technology could even allow something like transcendence)

**Comment by Badcholo** [comment]
My Modpack, technic launcher, Mojangsters has Hunger overhaul. My hunger drops 75% slower and you heal as long as you have at least 1 hunger bar. I have already fixed that problem with vanilla minecraft, so needing nutrition or a way to still use food or even possibly a way to auto feed yourself from internal or even inventory, as an upgrade would be a good option. The part of not needing oxygen anymore is understandable but maybe at the cost of an upgrade. Being an android should leave as much room as possible for upgrades not instantly give you all these benefits and make all food useless.

**Comment by shadoxxhd** [comment]
IMO the android transformation itself could be much more expensive, but needing oxygen as an android just makes no sense. if the borg transformation gets implemented, they would need oxygen (or whatever the pre-borgification species needs) before upgrading themselves with CO2 recycler/artificial lungs/electrolysis symbiont/whatever else their tech tree might feature.
that would make the borg/android distinction way bigger: while android transformations are quite high-tier and provide a big starting advantage which gets even bigger really fast when upgrading but have only access to purely synthetic components/upgrades, borg transformations are more like an infection in the beginning, providing only slight benefits (like 1% additional armor, .5% walk speed, 5% more badass look) at a high cost (nedding both energy and food to survive, can't swin properly due to higher weight, increased vulnerability vs some damage types, common 'blackout'/'pain confusion' screen effects, half of the screen constantly black due to not yet working eye implant, chance for nanobots going rogue and killing you (-> possible cross-mod interaction with open computers) and extreme difficulties to do anything while in the influence area of a borg queen with matching/compatible transmitter frequency (mind control like; if you don't press any key you will behave like any other NPC drone. harming the queen is completely impossible in this state (you'd rather kill yourself))). borg upgrades very expensive/complex, needing lots of resources, time and effort (1) to even reach the 'no advantages,no disadvantages' stage, but are far more potent than their android equivalent in the endgame (but require some orders of magnitude more investment). another advantage of being a borg is access to all tech trees, while androids and 'untransformed creatures' can only access 2 each ('android' + nano/cyber and nano/cyber + gen, respectively)(although specific, very late game techs from the nano/cyber tree can make most of the android tree available (while denying access to big parts of the bio tech tree - you can only be organic OR anorganic(or a combination, but eg. your left eye can't be both))) (borg have access to android, nano/cyber, bio and a special hivemind-themed borg tech tree - and while 'implicit transformations' through upgrading/exchanging body parts/... are possible, certain borg techs are infection-exclusive (some others could only be learned from cooperating with (or serving, if you are not a queen yet) another borg collective).


summary:
-the upgrade paths are not totally separated, but the normal transformations set you at different starting points in the tech tree (or rather tech web?)
-android: late-game transformation, relatively easy, well-documented (in-game/in-lore), quite powerful, simple and straight-forward
-human: normal, unaugmented starting point (you will be human when you spawn (or maybe select your race on first spawn?))
-alien: depends on race (2); different initial upgrade capabilities, different initial specs (although mostly low-tier)
-borg infection: mostly negative implants; however completely different starting point in the tech web
-borg assimilation: slightly better than infection, but a little more difficult to achieve (although both can happen 'by accident' (wrong place at the wrong time), the chance to be infected is much higher than that of an assimilation (infection happens by autonomous nanodrone swarms, assimilation by borg drone))
(difference between the last 2: expendable borg worker vs. slightly less expendable borg engineer)
-hybrid/transcendent life forms: only achievable through extensive research and self-modification (pure-bred androids will be unable to reach this stage (except for one way of mind uploading, which involves different tech trees though))
-perfect biotech: classification of extremely powerful life forms basing on sub cellular biotechnology (only achievable through most endgame borg technologies (sort of combining all the tech trees into one))(can overlap with transcendent category)


(1): very complicated research, much 'trial & error', requiring the breeding/construction/whatever of player-adapted solutions (step-by-step guides won't do it here), while android research is freely available on the in-game 'holonet' (you could just order your android limbs from some NPC) and other tech trees are between these extremes (nano/cyber is also quite straight-forward and modular, though it's more complicated than android and there are a few modules effecting each other (in a positive or negative way) while bio has considerable amounts of 'user specific' research/breeding/production and intermodular 'crosslinking' (although not nearly as dangerous and complicated as borg)(bio endgame tier technologies and borg mid-game and above ones could even incorporate designing new biochemical/bioelectrical/intrecellular/... interaction systems to further improve your overall performance. not enough defense? add nano-sized molecular integrity generators to your cells in specific patterns until you could survive the enterprise crashing on your head or incorporate light-bending nanostructures into your skin to be invisible and evade combat altogether)

(2): some race 'blueprints' (just random ideas)(all races also have distinct initial technology/ships/buildings):
'starting races':
-vulcans: higher intelligence -> some HUD helpers (ore highlighter, creeper-behind-your-back indicator), research bonus, better relations with thinking-type personality NPCs, worse relations with feeling-type personality NPCs, society: technocracy/communism techweb: cyber/nano and near pre-android
-klingons: base resistance, higher health, higher strength, worse relations except with races highly valuing honor, 'combat instincts' HUD/'autoaim' bonus, less initial augmentation potential (more resistant organs need more space etc.), society: aristocracy/traditionalism techweb: far pre-android ,some 'pre-installed' bio-improvements (but not researchable early-game)
-humans: above-average adaptability, can choose a starting trait (intelligence/dexterity/endurance)(smaller bonus than base race bonuses), weapon technology bonus (IRL we had fusion bombs before we started researching FTL travel...), society: manipulative,spying government (NSA etc)(pretended: democracy)/capitalism, techweb: cyber/nano
-(non-star trek) edenists: sub-society originated from humans, (relatively) heavily geneered (gen-engineered), telepathically (affinity), organic technology. stats human-like, society: 'consensus' (affinity-based democracy)/socialism, techweb starting point 80% bio 20% cyber/nano
endgame races:
-grey goo: mass of homogeneous cells, each able to perform any required function. 'limbs' can adopt any imaginable shape, from tentacles to arm-equivalents. massive adaptability, both spontaneous as well as bioengineered. can only be killed by destroying every living cell of it.
-wisp: self containing intelligent energy structure. can have a multitude of abilities and is invulnerable to most classical forms of damage (only vulnerable to certain quantum/subspace weapons (some may be vulnerable to EMPs or other electromagnetic weapons (including phasers))). may exist in multiple locations at the same time/phase into different dimensions/be able to distort spacetime at will.
-shapeShifter: life form that can energetically transform its appearance at will. individual forms usually less advanced and flexible than other endgame life forms, but dedicated to do a single task at maximum efficiency.
-replicator: specialized form of a shapeshifter that transforms inanimated matter into copies of itself, commonly working together as swarm intelligence (queen-drone models are also common, while completely individual clones are quite rare).

**Comment by simeonradivoev** [comment]
O.O you might be right on that last Edit. I simply don't have the time nor the resources to do that. It could be done if other people start making addons.

**Comment by shadoxxhd** [comment]
i think much of this is completely undoable in the current minecraft engine (alone due to it using java), but it could be a nice idea for a standalone RPG...

**Comment by Alexismachina** [comment]
shadoxxhd how long did it take to type ALL of this ? XD

**Comment by Badcholo** [comment]
omg i'm not reading all of that.

**Comment by shadoxxhd** [comment]
@Alexismachina idk, i guess it took me quite a while... xD

**Comment by Alexismachina** [comment]
@shadoxxhd Androids are not late game. Maybe even early game. RF is not something hard to make.
And tritanium is not the most uncommon thing in the world.

**Comment by shadoxxhd** [comment]
@Alexismachina they are not, but I think they should be.

**Comment by Alexismachina** [comment]
Heh, if they were end-game they should really be better.

**Comment by shadoxxhd** [comment]
@Alexismachina yes, they should.
